tests/unit_and_functional/test_cli_service_deployments.py

```diff
@@ -27,6 +27,20 @@
     display_service_deployments,
 )
 
+Test_ServiceStageDeployment = {
```

Comment:
Should probably be a fixture, or at the very least use camelcase for name if it isn't.

Reply:
Good spot, it was meant be camelcase but I put an underscore there by mistake out of force of habit from writing tests. 

I'll make it a fixture just in case we manipulate this object in future tests.

Reply:
Nice one 👍